### PR TITLE
EWPP-264: Add Address field facet processor.

### PR DESCRIPTION
**Pull request by imanoleguskiza**
### Description

Added a submodule that supports Address and Search API
### Change log

- Added: Added a submodule that supports Address and Search API
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by upchuk — CHANGES_REQUESTED** [review]
Shouldn't it be `oe_lists_pages_address`

File: modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+use CommerceGuys\Addressing\Country\CountryRepositoryInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\facets\FacetInterface;
+use Drupal\facets\Processor\BuildProcessorInterface;
+use Drupal\facets\Processor\ProcessorPluginBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Transforms the results to show the translated entity label.
```
Comment (upchuk) [inline modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php:13]: Update comment

File: modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ *
+ * @FacetsProcessor(
+ *   id = "format_address",
```
Comment (upchuk) [inline modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php:16]: Shou;dn't we namespace this ID with module name or something?

File: modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+ *   label = @Translation("Transform Address Codes into Full Names"),
```
Comment (upchuk) [inline modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php:17]: No need to capitalize every word :). Same for desc

File: modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+ *   description = @Translation("Transform Address Codes into Full Names."),
+ *   stages = {
+ *     "build" = 5
+ *   }
+ * )
+ */
+class FormatAddressProcessor extends ProcessorPluginBase implements BuildProcessorInterface, ContainerFactoryPluginInterface {
+
+  /**
+   * The subdivision repository.
```
Comment (upchuk) [inline modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php:27]: ?

File: modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+   *
+   * @var \CommerceGuys\Addressing\Country\CountryRepositoryInterface
+   */
+  protected $countryRepository;
+
+  /**
+   * Constructs a new object.
```
Comment (upchuk) [inline modules/oe_list_page_address/src/Plugin/facets/processor/FormatAddressProcessor.php:34]: a new FormatAddressProcessor

**Review by upchuk — CHANGES_REQUESTED** [review]

**Review by upchuk — APPROVED** [review]

**Review by hernani — APPROVED** [review]

**Pull request merged**